### PR TITLE
Change session difficulty icons to vector drawable

### LINKED ISSUE #397: [Assigned] Change session difficulty icons to vector drawable
## Overview (Required)
These icons in this PR.
https://github.com/DroidKaigi/conference-app-2018/pull/396

### PR DESCRIPTION
**Pull request by ycrack**
## Issue
- close #397 

## Overview (Required)
- Remove image binary
- Add vector image of level
- Change code to apply icons

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/item/SpeechSessionItem.kt [hunk 1]
```diff
@@ -35,9 +35,9 @@ data class SpeechSessionItem(
             //TODO: will implement this. Please check comments of issue #141
         }
         val levelDrawable = viewBinding.context.drawable(when (session.level) {
-            is Level.Beginner -> R.drawable.ic_beginner_lightgreen_20dp
-            is Level.IntermediateOrExpert -> R.drawable.ic_intermediate_senior_bluegray_20dp
-            is Level.Niche -> R.drawable.ic_niche_cyan_20dp
+            is Level.Beginner -> R.drawable.ic_beginner_lightgreen_20_dp
```
[the remark this answers is not in the document]
Comment (ycrack) [reply app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/item/SpeechSessionItem.kt]: Sure. Just renamed

**Comment by takahirom** [comment]
Thanks 👍
Probably you should create another branch instead of master. 
You should not commit to master, and you only use master for the pull.
And You create a branch by the master branch. and commit changes to created branch.

**Comment by takahirom** [comment]
We can squash merge. So we will do that 😄

**Comment by ycrack** [comment]
Thank you for suggestion! I am not good at using Github and Git, so I appreciate it. 😄 👍

**Pull request merged**